mac/macdeploy.sh

```diff
@@ -7,6 +7,7 @@
 
 MACDEPLOYQT="macdeployqt"
 LIBRARY_NAME="libQupZilla.2.dylib"
+PLUGINS="QupZilla.app/Contents/Resources/plugins"
```

Comment:
I find it strange that other plugins are correctly loaded from subdirectories, but `iconengines` not ...

**Docs**
During development, the directory for plugins is QTDIR/plugins (where QTDIR is the directory where Qt is 
installed), with each type of plugin in a subdirectory for that type, for example, styles. If you want your applications 
to use plugins and you don't want to use the standard plugins path, have your installation process determine the 
path you want to use for the plugins, and save the path, for example, by using QSettings, for the application to read when it runs. The application can then call QCoreApplication::addLibraryPath() with this path and your plugins will 
be available to the application. *Note that the final part of the path (for example, styles) cannot be changed.*

But if it works, then let's go with it.